ext/groonga/rb-grn-table.c

```diff
@@ -395,6 +400,8 @@ rb_grn_table_define_index_column (int argc, VALUE *argv, VALUE self)
                         "with_section", &rb_with_section,
                         "with_weight", &rb_with_weight,
                         "with_position", &rb_with_position,
+                        "index_small", &rb_index_small,
```

Comment:
Surely, it seems better to implement `:size => :small, :size => :medium` API. I've fixed.